src/engraving/layout/layoutcontext.h

```diff
@@ -69,6 +69,8 @@ class LayoutContext
     Fraction startTick;
     Fraction endTick;
 
+    double totalBracketsWidth = -1.0;
```

Comment:
Why isn't it 0 by default?

Reply:
We need to detect whether it had already been calculated or not. Even after it has been calculated, it may be 0, so we need to choose another distinguishable initial value.